src/utils/getMessage.js

```diff
@@ -0,0 +1,23 @@
+import languages from '../i18n';
+import getAttr from './getAttr';
+import config from '../config';
+
+/* eslint no-param-reassign: ["error", { "props": false }]*/
+
+/**
+ * Uses getAttr to obtain translations for UI message strings outside react-intl.
+ * @param  {string} string - The identifier for the string to be localized
+ * @param  {string} lang - Language code for requested language
+ */
+
+const getMessage = (string, lang) => {
+  const translations = config.languages.reduce((object, langCode) => {
```

Comment:
Can you explain what this reduce does (in a code comment)? Additionally, it looks like something that shouldn't be done on every invocation of `getMessage`.